toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -947,7 +947,7 @@ func InstallGrubEnv(installRoot string) (err error) {
 // - kernelCommandLine contains additional kernel parameters which may be optionally set
```

Comment:
We might want to add some details about `isBootParitionSeparate`.
also seems we have missed the `readOnlyRoot` last time.

Reply:
Added